cse/lang/__init__.py

```diff
@@ -47,9 +47,9 @@ def useCustomStopwordList(self, stopwords):
         return self
 
 
-    def appendToStopwordList(self, stopwords):
+    def appendToStopwordList(self, *stopwords):
         self.__useStopwords = True
-        self.__stopwords.append(stopwords)
+        self.__stopwords = self.__stopwords + list(stopwords)
```

Comment:
It's not directly related with the token position, is it?

Reply:
yeah. That was a bug in the previous code. (related to the mentioned caveates of dynamically typed languages)
`append` only accepts one element for the list
`+` allows concatenation of two lists

the method's definition does not specify explicitly what parameter type is needed, but the name of the method let someone assume a list is needed, therefore I changed it to handle both ways